lib/src/config_provider/config.dart

```diff
@@ -141,21 +136,12 @@ class Config {
         extractor: outputExtractor,
         extractedResult: (dynamic result) => output = result as String,
       ),
-      strings.headers: Specification<List<String>>(
-        description: 'List of C headers to generate bindings of',
+      strings.headers: Specification<Headers>(
+        description: 'Headers.',
```

Comment:
Are we exposing this description anywhere? (Like in `--help` or when the config is not valid?)

If yes, then maybe elaborate instead of making the description smaller.
If no, should we expose this documentation somewhere? Or should we remove it?

Reply:
Currently, it's not exposed anywhere, but we can add these to --help

Reply:
I am removing these descriptions for now.